fittings.gemspec

```diff
@@ -20,7 +20,9 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.required_ruby_version = ">= 2.4.0"
   s.add_dependency "hashie"
+  s.add_development_dependency "stitchfix-y"
```

Comment:
This process was probably different because it's public

Reply:
Reverted in e071985da27dd7715a20f86a8945312ae641fa86.